docs/plugins/mapper-annotated-text.asciidoc

```diff
@@ -18,7 +18,7 @@ include::install_remove.asciidoc[]
 [[mapper-annotated-text-usage]]
 ==== Using the `annotated-text` field
 
-The `annotated-text` tokenizes text content as per the more common `text` field (see 
+The `annotated-text` tokenizes text content as per the more common <<text, `text` field>> (see 
```

Comment:
Although the source files are in the same repo, this technically is a separate doc book. Across books, we have to use the external link syntax. This should fix the docs build error.
```suggestion
The `annotated-text` tokenizes text content as per the more common {ref}/text.html[`text`] field (see 
```